### PR TITLE
Phase II submission improvement

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM --platform=linux/amd64 ubuntu:20.04 as builder
```
Comment (xansec) [inline mayhem/Dockerfile:1]: Can we revisit the sed command here? Is there a better way to get this to build?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by rnshah9** [comment]
Still kind of a hack, but much less brittle I think.

File: wrapper.h [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// https://stackoverflow.com/questions/72177535/how-can-i-include-a-c-header-that-uses-a-c-keyword-as-an-identifier-in-c
```
Comment (xansec) [inline wrapper.h:1]: I guess this works.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**